crates/web-sys/Cargo.toml

```diff
@@ -1237,6 +1239,7 @@ TouchInit = []
 TouchList = []
 TrackEvent = ["Event"]
 TrackEventInit = []
+TransformStream = ["ReadableStream", "WritableStream"]
```

Comment:
Not having `WritableStream` available is breaking the build. Would you prefer `TransformStream` be moved in the next 'unstable' PR or those type included here?